src/plugins/workspace/public/components/workspace_list/index.tsx

```diff
@@ -130,47 +135,49 @@ export const WorkspaceList = () => {
       return '';
     }
 
-    return cleanWorkspaceId(
-      application.getUrlForApp(WORKSPACE_CREATE_APP_ID, {
-        absolute: false,
-      })
-    );
+    const appUrl = application.getUrlForApp(WORKSPACE_CREATE_APP_ID, {
+      absolute: false,
+    });
+    if (!appUrl) return '';
+
+    return cleanWorkspaceId(appUrl);
   }, [application, http]);
 
   const debouncedSetQueryInput = useMemo(() => {
     return debounce(setQueryInput, 300);
   }, [setQueryInput]);
 
-  const handleSearchInput: EuiSearchBarProps['onChange'] = ({ query }) => {
-    debouncedSetQueryInput(query?.text ?? '');
-  };
+  const handleSearchInput: EuiSearchBarProps['onChange'] = useCallback(
+    ({ query }) => {
+      debouncedSetQueryInput(query?.text ?? '');
+    },
+    [debouncedSetQueryInput]
+  );
 
   const search: EuiSearchBarProps = {
     onChange: handleSearchInput,
     box: {
       incremental: true,
     },
     toolsRight: [
-      <EuiButton href={workspaceCreateUrl} key="create_workspace">
+      <EuiButton
+        href={workspaceCreateUrl}
+        key="create_workspace"
+        data-test-subj="workspaceList-create-workspace"
+      >
         Create workspace
       </EuiButton>,
     ],
   };
 
-  const selectionValue: EuiTableSelectionType<WorkspaceAttribute> = {
-    selectable: () => true,
-    onSelectionChange: (selection) => {
-      setSelection(selection);
-    },
-  };
-
   return (
     <EuiPage paddingSize="none">
       <EuiPageBody panelled>
         <EuiPageHeader
           restrictWidth
           pageTitle="Workspaces"
           description={WORKSPACE_LIST_PAGE_DESCRIPTIOIN}
+          style={{ paddingBottom: 0, borderBottom: 0 }}
```

Comment:
Can we control the style with `paddingSize` and `bottomBorder` props of `EuiPageHeader`?

Reply:
I tried to use `bottomBorder` before but it doesn't work.